src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/test/TestBase.java

```diff
@@ -195,9 +216,70 @@ protected void quit() {
 		}
 	}
 	
+	protected WebElement getElement(By elementBy) {
+		try {
+			return driver.findElement(elementBy);
+		} catch (NoSuchElementException e) {
+			return null;
+		}
+	}
+
+	protected String trimPatientId(String id) {
+		id = id.replace("Recent", "");
+		if (id.indexOf("[") > 0) {
+			id = id.split("\\[")[0];
+		}
+		if (id.indexOf(" ") > 0) {
+			id = id.split(" ")[0];
+		}
+		return id;
+	}
+	
+	protected void matchPatientIds(String patientId) {
+		List<String> ids = new ArrayList<>();
+		driver.findElements(patientHeaderId).forEach(id-> {
+			ids.add(trimPatientId(id.getText()));
+		});
+		assertTrue(ids.contains(trimPatientId(patientId)));
+	}
+	
 	public String getCurrentDate() {
 		Date date = new Date();
 		SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yyyy");
 		return formatter.format(date);
 	}
+	
+	protected void initiateWithLogin() {
```

Comment:
we dont need this util , the function already exist in the Login page [here](https://github.com/IsantePlus/isanteplus-qaframework/blob/main/src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/page/LoginPage.java#L71-L78)

Reply:
Resolved thanks